Desktop/analysis/analysis.cpp

```diff
@@ -1022,6 +1022,17 @@ void Analysis::analysisQMLFileChanged()
 		Log::log() << "Form (" << form() << ") wasn't complete " << ( form() ? std::to_string(form()->formCompleted()) : " because there was no form...") << " yet, and also did not have a QML error set yet, so ignoring it." << std::endl;
 }
 
+void Analysis::generateFileWrapper()
+{
+   QString wrapper = form()->generateWrapper();
```

Comment:
Maybe add a check *inside* this function whether the analysis in question is actually the dev module

Reply:
This function is not called with a dev module.

Reply:
If it isnt called with a devModule why are you then using `devModSourceDirectory` ???